benchcab/data/test/integration.sh

```diff
@@ -20,8 +20,12 @@ project: $PROJECT
 experiment: AU-Tum
 
 realisations:
-  - path: trunk
-  - path: branches/Users/sb8430/test-branch
+  - repo:
+      svn:
+        branch_path: trunk
+  - repo:
+      svn:
+        branch_path: branches/Users/sb8430/test-branch
```

Comment:
Depends on how long we are looking at maintaining the SVN version. If there is sufficient cutover time then it might pay to do tests on both, but if we want to move on from SVN we can start deprecating support now.

Reply:
I would probably update to git now.
@SeanBryan51 is it worth adding the results of an integration test with both git and svn to this PR running `benchcab build` ? This should cover the bulk of what changed.

Reply:
I don't see any issues with using an SVN and git repo in the integration test. We could do the following
```suggestion
  - repo:
      svn:
        branch_path: trunk
  - repo:
      git:
        branch: main
```

Reply:
> I would probably update to git now. @SeanBryan51 is it worth adding the results of an integration test with both git and svn to this PR running `benchcab build` ? This should cover the bulk of what changed.

Sure